### PR TITLE
website: add pricing waitlist and blog post

### PR DESCRIPTION
**Pull request by BeryJu**

### CODE REVIEW

File: website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+One of the primary ways for us to hear your input will be right there on the new page. We have created a “waitlist” for each pricing plan, in order to gauge interest in the plans’ feature sets and to learn what feedback you have on the pricing in general. Please join the list and the conversation!
+
+### About the plans
+
+The following offerings are described in detail on the new page (coming soon!) in our website.
+
+-   Open Source
+
+    Our forever-free offering, the open source authentik has been active for over 5 years, and now has the full sppor
```
[the remark this answers is not in the document]
Comment (tanberry) [reply website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md:67]: No actually I cut myself off mid-sentence to go fight with Docker, lol

File: website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+
+-   Enterprise Self-hosted
+-   Enterprise Cloud
```
Comment (tanberry) [inline website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md:70]:
```suggestion
-   Enterprise Cloud
   The Enterprise Cloud plan provides the convenience of our enterprise-level product as a SaaS offering, hosted and managed by Authentik Security. For many organizations, the benefits of decreased operational costs and universal data access (no VPN, servers, and network configuration required) make SaaS the best choice. With the cloud offering, the same enterprise-level support plan is included, and migrating to self-hosted is always an option.
```

File: website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md [hunk 3]
```diff
@@ -0,0 +1,74 @@
...
+      url: https://github.com/tanberry
+      image_url: https://github.com/tanberry.png
+tags:
+    - announcement
+hide_table_of_contents: false
+---
+
+In a blog from last November 2022, titled “[Next steps for authentik](https://goauthentik.io/blog/2022-11-02-the-next-step-for-authentik)”, I wrote about the launch of [Authentik Security](https://goauthentik.io/), our open core company built around the open source project [authentik](https://github.com/goauthentik/authentik).
+
+In this post, we’d like to provide updates on our progress in building out Authentik Security the company, ramping up the feature set in our open source identity provider, and taking the first steps in developing/offering an enterprise-level feature set for the Cloud or self-hosting. We are enthusiastic about our path forward and our plans to take authentik from a project to a product.
```
Comment (tanberry) [inline website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md:20]:
```suggestion
In this post, we’d like to provide updates on our progress in building out Authentik Security the company, ramping up the feature set in our open source identity provider, and taking the first steps in developing and offering an enterprise-level feature set for the Cloud or self-hosting. We are enthusiastic about our path forward and our plans to take authentik from a project to a product.
```

File: website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+
+<!--truncate-->
+
+## Company Updates
+
+Authentik Security is now officially almost 6 months old. The energy and vitality of our community-based authentik project continues, and now with a growing staff we can keep the open source product expanding with new releases and also work on launching and growing the company. Mixed into that balance is defining our feature sets for each plan, and developing new functionality for companies that want to deploy authentik at the enterprise level, either self-hosted or on the Cloud as a Saas offering by Authentik Security.
```
Comment (tanberry) [inline website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md:26]:
```suggestion
Authentik Security is now officially almost 6 months old. The energy and vitality of our community-based authentik project continues, and now with a growing staff we can keep the open source product expanding with new releases and also work on launching and growing the company. Mixed into that balance is defining our feature sets for each of our offerings, and developing new functionality for companies that want to deploy authentik at the enterprise level, either self-hosted or on the Cloud as a SaaS offering by Authentik Security.
```

File: website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md [hunk 5]
```diff
@@ -0,0 +1,74 @@
...
+
+We have a lot of exciting work in front of us. Check out our [job posts](https://goauthentik.io/jobs/) and consider joining us!
+
+## New features for authentik
+
+Authentik Security aims to always be open and transparent, and our trust in our community’s awesomeness means that we realize you all are experts in the field, the ones working in the security and identity management industry every day (and night!), so we look forward to strengthening our collaboration and communication with you, our user.
+
+We have a roadmap with several new features, and we want to hear your opinions on them. You can write us directly at hello@goauthentik.io or open a GitHub issue. For more targeted conversations, we will reach out to schedule calls with users who want to provide more in-depth collaboration and feedback.
```
Comment (tanberry) [inline website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md:34]: @BeryJu  do you think we should use a mail:to link for the email address?

Comment (tanberry) [inline website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md:34]:
```suggestion
We have a roadmap with several new features, and we want to hear your opinions on them. You can write us directly at hello@goauthentik.io or open a [GitHub issue](https://github.com/goauthentik/authentik/issues). For more targeted conversations, we will reach out to schedule calls with users who want to provide more in-depth collaboration and feedback.
```

**Review by tanberry — COMMENTED** [review]
Sorry it took so long! I think after you review one more time, it should be good to go.

File: website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md (hunk 5 above)
Comment (BeryJu) [reply website/blog/2023-03-23-whats-new-with-authentik-march-2023/item.md:34]: Docusaurus seems to do this automatically

**Pull request merged**